File: Modul 6/SOAL-1/PRAK601_2410817220015_Ashma_Salimah.c
#include <stdio.h>
int main() {
    int baris, kolom;
    scanf("%d %d", &baris, &kolom);
    int matriks[baris][kolom], i, j;
    for(i = 0; i < baris; i++) {
        for(j = 0; j < kolom; j++) {
            scanf("%d", &matriks[i][j]); } }
    printf("\n");
    for(i = 0; i < baris; i++) {
        for(j = 0; j < kolom; j++) {
            printf("%d ", matriks[i][j]); } 
        printf("\n"); }
    return 0;
}

File: Modul 6/SOAL-1/PRAK601_2410817220015_Ashma_Salimah.py
baris, kolom = map(int,input("").split())
angka = list(map(int,input("").split()))
matriks = []
for i in range (baris) :
    matriks.append(angka[i * kolom:(i + 1) * kolom])
print("")
for i in range (baris) :
    for j in range (kolom) :
        print(f"{matriks[i][j]}", end=" ")
    print("")